### ISSUE TITLE
Hey.

### ISSUE THREAD

**Issue by Time-dragon** [Time-dragon/tw-build#2]
@GarboMuffin I never sent you any emails. I don't know your email in the first place. I also make USEFUL PRs.

**Comment by Time-dragon** [comment]
So unblock me.

**Comment by GarboMuffin** [comment]
Hi Time-Dragon,

The email from GitHub is not the primary reason you are blocked. Yes, it came from "github.com", but it was sent because you invited me to a spam organization. It doesn't matter anyways. The big issue is that you sent five invalid ("spam") pull requests (https://github.com/TurboWarp/scratch-gui/pulls?q=is%3Apr+author%3ATime-dragon) despite several requests to stop (https://github.com/TurboWarp/scratch-gui/pull/218#issuecomment-853288766, https://github.com/TurboWarp/scratch-gui/pull/228#issuecomment-855467896, https://github.com/TurboWarp/scratch-gui/pull/229#issuecomment-855468218) and at least a couple temporary blocks. One or two invalid pull requests was not itself cause for an indefinite block: the problem is that you kept making them after being asked to stop. It really is a shame because some of your issues were valid, but for some reason you couldn't stop opening invalid pull requests.

My offer to reconsider if you send a well-written email to contact@turbowarp.org acknowledging your mistakes and what you will do to prevent it from happening again is still open. If you don't do that then this is the last message you will receive from me.

Good luck with your other endeavors.

**Comment by Time-dragon** [comment]
> you invited me to a spam organization

No. It was forkforkrus, a fork of forkphorus that is two times as fast.
